Lib/bdb.py

```diff
@@ -26,6 +34,12 @@ def __init__(self, skip=None):
         self.frame_returning = None
 
     def canonic(self, filename):
+        """Get a filename into a canonical form.
```

Comment:
https://bugs.python.org/issue477023 works.  The bracketed path names are from interactive mode.